gerrit-start.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env sh
 set -e
-#TODO:Not sure if gerrit can be stopped properly...
```

Comment:
According to the log output, the container stop properly:
````
[2017-08-23 08:22:23,110] [main] INFO  com.google.gerrit.pgm.Daemon : Gerrit Code Review 2.14.2 ready
[2017-08-23 08:22:31,836] [ShutdownCallback] INFO  com.google.gerrit.pgm.Daemon : caught shutdown, cleaning up
[2017-08-23 08:22:31,860] [ShutdownCallback] INFO  org.eclipse.jetty.server.AbstractConnector : Stopped ServerConnector@f9cd1e6{HTTP/1.1,[http/1.1]}{0.0.0.0:8080}
[2017-08-23 08:22:31,865] [ShutdownCallback] INFO  org.eclipse.jetty.server.handler.ContextHandler : Stopped o.e.j.s.ServletContextHandler@35f6f105{/,null,UNAVAILABLE}
[2017-08-23 08:22:31,893] [ShutdownCallback] INFO  com.google.gerrit.sshd.SshDaemon : Stopped Gerrit SSHD
[2017-08-23 08:22:31,923] [ShutdownCallback] INFO  com.google.gerrit.server.plugins.PluginLoader : Unloading plugin events-log, version v2.13-26-g83e640c
[2017-08-23 08:22:31,926] [ShutdownCallback] INFO  com.google.gerrit.server.plugins.PluginLoader : Unloading plugin delete-project, version v2.13-19-g503e16b
[2017-08-23 08:22:32,178] [ShutdownCallback] INFO  com.google.gerrit.server.plugins.PluginLoader : Cleaned plugin plugin_events-log_170823_0822_7831261295650918664.jar
[2017-08-23 08:22:32,178] [ShutdownCallback] INFO  com.google.gerrit.server.plugins.PluginLoader : Cleaned plugin plugin_delete-project_170823_0822_5862437198760081076.jar
````
So we can safely remove the line indeed.